src/calliope_pathways/models/italy_stationary/model_config/techs.yaml

```diff
@@ -0,0 +1,305 @@
+    lifetime: 35
+
+    # IEA
+    cost_flow_cap.data: 490 # EUR2010/kW, assumed -> overnight OCGT plant median
+    # Lombardi
+    cost_flow_in.data: 0.01 # EUR2010/kW
+    cost_flow_out.data: 0.023 # EUR2010/kWh
+    cost_om_annual.data: 32 # EUR2010/kW/yr
+
+  coal:
+    name: 'Coal plant'
+    color: '#5A5A5A'
+    base_tech: supply
+    inherit: cost_dim_setter
+    carrier_out: power
+    flow_out_eff: 0.38
+    flow_ramping: 0.6
+    lifetime: 35
+
+    # IEA
+    cost_flow_cap.data: 1584 # EUR2010/kW, overnight median
+    # Lombardi
+    cost_flow_in.data: 0.00946 # EUR2010/kW
+    cost_flow_out.data: 0.023 # EUR2010/kWh
+    cost_om_annual.data: 32 # EUR2010/kW/yr
+
+  ccgt:
+    name: 'Combined cycle gas turbine'
+    color: '#A5A5A5'
+    base_tech: supply
+    inherit: cost_dim_setter
+    carrier_out: power
+    flow_out_eff: 0.56
+    flow_ramping: 0.8
+    lifetime: 20
+
+    # IEA
+    cost_flow_cap.data: 709 # EUR2010/kW, overnight median
+    # Lombardi
+    cost_flow_in.data: 0.0291 # EUR2010/kWh
+    cost_flow_out.data: 0.00315 # EUR2010/kWh
+    cost_om_annual.data: 10.5 # EUR2010/kW/yr
+
+  wind:
+    name: 'On-shore wind power'  # TODO: fix source
+    color: '#47D154'
+    base_tech: supply
+    inherit: cost_dim_setter
+    carrier_out: power
+    source_unit: per_cap
+    lifetime: 20
+
+    # RSE, 2015 values
+    cost_flow_cap.data: 1451 # EUR2010/kW
+    cost_om_annual.data: 38 # EUR2010/kW/yr
+
+  pv:
+    name: "Photovoltaic power"  # TODO: fix source
+    color: '#DDCC77'
+    base_tech: supply
+    inherit: cost_dim_setter
+    carrier_out: power
+    source_unit: per_cap
+    lifetime: 22
+
+    # from RSE, 2015 estimate
+    cost_flow_cap.data: 1322 # EUR2010/kW
+    cost_om_annual.data: 24 # EUR2010/kW/yr
+
+  battery_li:
+    name: "Li-ion battery storage"
+    color: '#882255'
+    base_tech: storage
+    inherit: cost_dim_setter
+    carrier_in: power
+    carrier_out: power
+    flow_cap_max: # kW
+    storage_cap_max: 2e8
+    flow_cap_per_storage_cap_max: 0.25
+    flow_out_eff: 0.98
+    flow_in_eff: 0.98
+    lifetime: 15
+
+    flow_cap_initial: 0
+
+    # Lombardi
+    cost_storage_cap.data: 433 # EUR2010/kWh storage capacity
+    cost_om_annual.data: 7.22 # EUR2010/kW/yr
+    cost_flow_out.data: 0.0022 # EUR2010/kWh
+
+  battery_phs:
+    name: "Pump-hydro storage"
+    color: "#332288"
+    base_tech: storage
+    inherit: cost_dim_setter
+    carrier_in: power
+    carrier_out: power
+    flow_cap_max: # kW
+    flow_out_eff: 0.97
+    flow_in_eff: 0.87
+    lifetime: 50
+
+    # RSE, average of 2020 (no 2015 data)
+    cost_storage_cap.data:  3218 # EUR2010/kWh storage capacity
+    cost_om_annual.data: 45 # EUR2010/kW/yr
+
+  ##
+  # Demand, imports, exports
+  ##
+  demand_power:
+    name: "Power demand"
+    color: "#072486"
+    base_tech: demand
+    carrier_in: power
+
+  import_power:
+    name: "Power imports"
+    color: '#E68A31'
+    base_tech: supply
+    carrier_out: power
```

Comment:
I prefer "electricity" to "power" as a carrier, although this is a deviation from the original model